demo_bevy/Cargo.toml

```diff
@@ -18,10 +18,9 @@ steam = ["bevy_renet/steam", "steamworks"]
 bevy_rapier3d = "0.22.0"
 bevy = {version = "0.11", default-features = false, features = ["bevy_core_pipeline", "bevy_render", "bevy_asset", "bevy_pbr", "x11", "tonemapping_luts", "ktx2", "zstd"]}
 bevy_renet = { path = "../bevy_renet", features = ["serde"] }
-serde = { version = "1.0", features = [ "derive" ] }
+serde = { version = "1.0", features = ["derive"] }
 bincode = "1.3.1"
-# Use version directly when egui is updated to 0.22
```

Comment:
I did not completely grasp what this comment means - so I just updated to bevy_egui 0.22 which (counter intuitively) uses egui 0.23

Reply:
I forgot to remove this comment :0